test-one-stone-at-a-time.sh

```diff
@@ -62,4 +85,6 @@ sleep 10
 run_white_stone &
 sleep 10
 
-echo "done" >&2
+echo "Finished" >&2
+echo "Stop the hardware" >&2
```

Comment:
Backgrounding of start_hardware & stop_hardware is indeed not necessary in current implementation. But it is for run__stone, as otherwise the functions prevent the toplevel sleeps between the different stone color from starting, which makes it impossible to have multiple stones in the simulation at the same time.
Backgrounding for start_hardware & stop_hardware was just provided, to have a consistent behaviour, if for example sleeps would be added in those two functions.